_observing-your-data/alerting/comp-monitors.md

```diff
@@ -0,0 +1,513 @@
+| Audit alert | Delegate monitors generate alerts in an **audit** state. Users are not notified about each individual audit alert and don't need to acknowledge them. Audit alerts are used to evaluate chained alert trigger conditions in composite monitors. |
+| Execution | A single run of all delegate monitors in the sequence defined in the composite monitor's configuration. |
+| Execution Id | Allows for the management of data recorded from a specific execution of a composite monitor. The execution Id associates findings and alerts with the execution and is stored in each monitor's metadata, along with the workflow Id and the monitor Id. |
+
+
+## Example workflows
+
+In a composite monitor, the chained outputs from the individual delegate monitors can be either findings or alerts. The following sections describe the workflow for each and provide an example of how they work.
+
+
+### Chained findings
+
+As an example of chained findings, consider a composite monitor configured with two delegate monitors. The second monitor is defined to use findings from the first as its input.
+
+* The first delegate monitor (monitor #1) is a per document monitor configured to analyze a data source with the following three queries:
+   
+   1. Request payload size greater than 100 Kb.
+   2. Response status = 200.
+   3. Response contains a specific header.
+   
+   For every execution, the monitor queries the data source and generates findings for documents that match the conditions set out by the queries. These results by themselves are valuable. However, by adding a second monitor the sequence can apply further analysis to events in the data source.
+* The second monitor (monitor #2) is a per bucket monitor that aggregates data by client IP, checks how many IPs there are, and determines how many IPs are sending these types of requests. The composite monitor configuration provides a way to add the second monitor in sequence so that it executes following the first.
+* Monitor #2 first filters the findings from Monitor #1 and then queries the data derived from monitor #1. Matches with this second set of queries then generate triggers and alerts for notifications.
+* Both monitors #1 and #2 return trigger results to the composite monitor according to their sequence in the configuration. The composite monitor returns a list of trigger results from monitors #1 and #2.
+
+The following image shows a simplified workflow for a composite monitor with chained findings.
+
+{% include gif-pause.html %}
+
+
+### Chained alerts
+
+As an example of chained alerts, consider a composite monitor configured with three delegate monitors. The first and second monitors (monitors #1 and #2) are configured to generate audit alerts when two specific events happen across the cluster. A third monitor (monitor #3) is configured to send an alert when both monitors #1 and #2 generate their own alerts.
```

Comment:
The functionality has changed
now the functionality in example would be like
workflow execution starts -> execute delegate monitor 1 -> get its result. execute delegate monitor 2 -> get its result. -> execute chained alert triggers. -> send out chained alert notifications